yucca/image_processing/transforms/normalize.py

```diff
@@ -0,0 +1,30 @@
+from yucca.image_processing.transforms.YuccaTransform import YuccaTransform
+from yucca.preprocessing.normalization import normalizer
+
+
+class Normalize(YuccaTransform):
```

Comment:
And this might, as I see it, not support all normalization modes at the moment (e.g. the ones that rely on dataset stats). This is fine, you dont have to implement that right now, because there'll be some implementing to do there, with how we transport that information to the augmentation pipeline. But can you maybe add this information as a comment in this class, so for future we dont forget this?

Reply:
Even better, i'll add it as an assert!